src/pages/index.js

```diff
@@ -14,70 +14,176 @@ import {
   inputJob,
 } from "../utils/constants.js";
 import "./index.css";
+import Api from "../components/Api.js";
+import PopupWithVerify from "../components/PopupWithVerify.js";
 
+//Api
+const api = new Api("https://around-api.en.tripleten-services.com/v1", {
+  authorization: "57aa7463-f839-44ea-835b-493cff8bea63",
+  "Content-Type": "application/json",
+});
+
+api
+  .getInitialCards()
+  .then((res) => {
+    res.forEach((card) => {
```

Comment:
Needs correction: You should use renderItems method of the Section class to render the initial cards